TestJava/src/main/java/ro/siit/domain/ShapeService.java

```diff
@@ -0,0 +1,44 @@
+package ro.siit.domain;
+
+import java.util.List;
+
+public class ShapeService {
+    public double totalPerimeter(List<Shape> shapes) {
+        double perimeter = 0;
+
+        for (Shape shape : shapes) {
+            perimeter = shape.perimeter();
+        }
+        return perimeter;
+    }
+
+    public double totalSurface(List<Shape> shapes) {
+        double surface = 0;
+
+        for (Shape shape : shapes) {
+            surface = shape.surface();
+        }
+        return surface;
+    }
+
+    public boolean similarShapes(Rectangle rectangle1, Rectangle rectangle2){
+        if (rectangle1.perimeter() == rectangle2.perimeter()){
```

Comment:
you could have a square and a rectangle with the same permiter and surface. You need to also check that the shape type is the same. (e.g. test first that you compare two rectangles, squares, etc)